client/src/pages/AssignRoles.js

```diff
@@ -9,6 +9,11 @@ import AvailableRoles from '../components/AvailableRoles';
 import { MatchContext } from '../ContextProvider/match';
 import io from "socket.io-client";
 
+import Popup from 'reactjs-popup';
```

Comment:
Material UI has a modal component couldn't that be used instead of importing another library?

https://material-ui.com/components/modal/

Reply:
Finished replacement